modules/apps/calendar/calendar-web-test/src/testFunctional/tests/CalendarConfiguration.testcase

```diff
@@ -60,9 +60,64 @@ definition {
 	@ignore = "Test Stub"
 	@priority = "3"
 	test CanConfigureEventsDefaultDuration {
+		CalendarNavigator.gotoAddEvent();
+
+        CalendarEvent.editTitle(eventTitle = "Calendar Event Title");
+
+        CalendarEvent.setTimes(
+            endDateTime = "02:00 PM",
+            startDateTime = "01:00 PM");
+
+        CalendarEvent.publish();
+
+        CalendarScheduler.selectSpecificEvent(eventTitle = "Calendar Event Title");
+
+        CalendarScheduler.gotoDetailsViaDialogBox();
+
+        CalendarEvent.viewTimesViaDetails(
+            endDateTime = "02:00 PM",
+            startDateTime = "01:00 PM");
+
+        Navigator.gotoPage(pageName = "Calendar Page");
+
+        CalendarNavigator.gotoConfiguration();
+
+        CalendarConfiguration.viewSelectDefaultDuration(defaultDuration= "60 Minutes");
+
+        CalendarConfiguration.selectDefaultDuration(defaultDuration = "15 Minutes");
+
+        CalendarConfiguration.save();
+
+        Alert.viewSuccessMessage();
+
+        Navigator.gotoPage(pageName = "Calendar Page");
+
+        CalendarNavigator.gotoAddEvent();
+
+        CalendarEvent.editTitle(eventTitle = "Calendar Event Title 2");
+
+        CalendarEvent.setTimes(
+            endDateTime = "03:15 PM",
+            startDateTime = "03:00 PM");
+
+        CalendarEvent.publish();
+
+		Navigator.gotoPage(pageName = "Calendar Page");
+		
+        CalendarScheduler._selectSpecificEventCurrentDay(eventTitle = "Calendar Event Title 2");
+
+        CalendarScheduler.gotoDetailsViaDialogBox();
+
+        CalendarEvent.viewTimesViaDetails(
+            endDateTime = "03:15 PM",
+            startDateTime = "03:00 PM");
+
+        Navigator.gotoPage(pageName = "Calendar Page");
 
-		// TODO LRQA-71621 CanConfigureEventsDefaultDuration pending implementation
+        CalendarNavigator.gotoConfiguration();
 
+        CalendarConfiguration.viewSelectDefaultDuration(defaultDuration = "15 Minutes");
```

Comment:
In this test, you are manually entering the desired time and the time space that we are trying to verify(15, 45, 60 minutes). You can look for a way to insert the event without directly interfering with the time, being able to just check if the default time was modified during the test. 
You can take a look at the macro 
```
CalendarScheduler.addLocalizedEvent(
			eventTitle = "Calendar Event Title",
			saveButton = "Save",
			viewType = "Week");
```